Userland/Libraries/LibC/inttypes.h

```diff
@@ -15,7 +15,7 @@ __BEGIN_DECLS
 #ifdef __i386__
 #    define __PRI64_PREFIX "ll"
 #    define __PRIPTR_PREFIX
-#else
+#elif __x86_64__
```

Comment:
This one and the previous one in LibC do hold up. Aarch64 and x86_64 are both 64-bit.

Reply:
👍